### ISSUE TITLE
About hyperparameters in CUB dataset

### ISSUE THREAD

**Issue by yyyxcleo** [linlany/RAPF#10]
Hii,thanks for your great job!

I encountered some issues while migrating the CUB dataset. I used the same hyperparameters as imagenet-r, but during training on Task 1, the loss switches to nan, and an error occurs at the end. I would like to ask whether this is a hyperparameter issue and if you could provide recommended hyperparameter values for CUB.

thx a lot!

![Image](https://github.com/user-attachments/assets/647fcb8a-7b7b-4a4a-8b5e-ae11f75c6004)

![Image](https://github.com/user-attachments/assets/a0747fc0-c5ba-4e67-9667-321aae542cf3)

![Image](https://github.com/user-attachments/assets/17a5551e-8efa-4cb8-93e2-daf2b11979e0)

**Comment by linlany** [comment]
The CUB-related code is uploaded now. Based on local testing, no issues have been found so far. Since a considerable amount of time has passed, I'm not sure what might have caused your issue.
